services/core-web/src/components/common/CoreTable.tsx

```diff
@@ -2,14 +2,14 @@ import React from "react";
 import { Table, TableProps, Tooltip } from "antd";
 import { ColumnsType } from "antd/es/table";
 import { MinusSquareFilled, PlusSquareFilled } from "@ant-design/icons";
-import { ExpandableConfig } from "antd/lib/table/interface";
+import { ExpandableConfig, TableRowSelection } from "antd/lib/table/interface";
 
 interface CoreTableExpandConfig<T> extends ExpandableConfig<T> {
   getDataSource?: (record: T) => any[];
   rowKey?: string | ((record: any) => string);
   recordDescription?: string;
   subTableColumns?: ColumnsType<any>;
-  matchChildColumnsToParent?: boolean;
+  showVersionHistory?: boolean;
```

Comment:
I don't think we should change the prop name here from matchChildColumnsToParent --> showVersionHistory, that should stay on DocumentTable.

With document table, there are other UI changes that are very specific to showing the version history (like the version tags, etc) so that is specifically what that prop is for. But the CoreTable doesn't just show documents, we want a prop name that makes sense here for any scenario that can have matching parent/child columns.

Reply:
That makes sense, the areas that had matchChildColumnsToParent changed are now reverted back to including it.